client/my-sites/email/titan-setup-mailbox/titan-setup-mailbox-form.jsx

```diff
@@ -183,7 +183,7 @@ const TitanSetupMailboxForm = ( { areSiteDomainsLoaded, selectedDomainName } ) =
 					busy={ isBusy }
 					onClick={ handleSetup }
 				>
-					{ translate( 'Set Up' ) }
+					{ translate( 'Complete Setup' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **31** times:
`translate( 'Complete setup' )` *ES Score: 10*